package.json

```diff
@@ -1,14 +1,18 @@
 {
-  "name": "zoottelkeeper-obsidian-plugin",
-  "version": "0.6.0",
-  "description": "Zoottelkeeper creates and maintains index files in all of your folders in your vault. If you create/delete/move a note, the index files will be updated automatically.",
+  "name": "obsidian_autoMOC",
+  "version": "0.0.2",
+  "description": "This plugin automatically creates, maintains and tags MOCs for all your folders. It is based on Zoottelkeeper 0.6.0.",
   "main": "main.js",
   "scripts": {
     "dev": "rollup --config rollup.config.js -w",
     "build": "rollup --config rollup.config.js --environment BUILD:production"
   },
-  "keywords": ["zettelkasten", "obsidian.md", "obsidian-plugin"],
-  "author": "Akos Balasko",
+  "keywords": [
+    "zettelkasten",
+    "obsidian.md",
+    "obsidian-plugin"
+  ],
+  "author": "Akos Balasko, Micha Brugger",
   "license": "MIT",
   "devDependencies": {
     "@rollup/plugin-commonjs": "^18.0.0",
```

Comment:
could you please remove the hats(^) from the dependency versions? If we specify precisely the required versions of the dependencies, we prevent us from break our stuff by some bugs may be added in the upcoming versions there.

Reply:
I will do that! Once again, I'm new to this and never did any "professional" programming, so I really appreciate when you just let me know whenever you notice things like that, I'll try to keep them in mind for next time 👍🏼

Reply:
No worries :)